File: backend/scripts/dump_users.js
const mongoose = require('mongoose');

async function run() {
  await mongoose.connect('mongodb+srv://LamNguyen:NTL123456@cluster0.zoxsh.mongodb.net/NTL_BinhThuong?retryWrites=true&w=majority', { useNewUrlParser: true, useUnifiedTopology: true });
  const users = await mongoose.connection.collection('users').find({}).toArray();
  for (const u of users) {
    console.log(`User ${u.phone} / ${u.fullName}: FCM: ${u.fcmToken}`);
  }
  process.exit(0);
}
run();

File: dump_users.py
from pymongo import MongoClient
import certifi

try:
    client = MongoClient('mongodb+srv://LamNguyen:NTL123456@cluster0.zoxsh.mongodb.net/NTL_BinhThuong?retryWrites=true&w=majority', tlsCAFile=certifi.where())
    db = client.get_database()
    users = list(db.users.find())
    for u in users:
        print(f"User {u.get('phone')} / {u.get('fullName')}: FCM: {u.get('fcmToken')}")
except Exception as e:
    print('Error:', e)
